common/src/KokkosFFT_Helpers.hpp

```diff
@@ -39,10 +38,10 @@ auto _get_shift(const ViewType& inout, axis_type<DIM> _axes,
 }
 
 template <typename ExecutionSpace, typename ViewType>
-void _roll(const ExecutionSpace& exec_space, ViewType& inout,
-           axis_type<1> shift, axis_type<1>) {
+void roll_impl(const ExecutionSpace& exec_space, ViewType& inout,
+               axis_type<1> shift, axis_type<1>) {
```

Comment:
```suggestion
void roll(const ExecutionSpace& exec_space, ViewType& inout,
               axis_type<1> shift, axis_type<1>) {
```

Why add `_impl` as you are already in an `Impl` namespace?